components/style/values/computed/image.rs

```diff
@@ -54,6 +54,8 @@ pub enum LineDirection {
     Angle(Angle),
     /// A corner.
     Corner(X, Y),
+    /// A Position and an Angle for legacy `-moz-` prefixed gradient.
+    PositionAngle(Option<Position>, Option<Angle>),
```

Comment:
Yes, I don't think we should parse `-moz` gradients without a clear reason to. We already fail to parse other stuff like `@-moz-keyframes`.